### DISCUSSION TITLE
Doesn't get load event work on images

### DISCUSSION THREAD

**Discussion by gregorvoinov** [nuxt/nuxt#15911] [Questions]
Hi,

what I'm doing wrong? I want to fire the load event on images, but it doesn't work

https://codesandbox.io/s/hardcore-glitter-lqb5uu?file=/app.vue

```js
<template>
  <img
    src="https://nuxt.studio/landing.png"
    @load="onLoad"
    style="width: 50%"
  />
</template>

<script>
export default {
  setup() {
    const onLoad = (e) => {
      console.log("looooaded", e);
    };
    return {
      onLoad,
    };
  },
};
</script>
```

**Comment by TheAlexLichter** [comment]
Stackblitz/CodeSandbox has problems with images + CORS, thus the `load` does not trigger. But the events work - you can test it via `@error` ☺️

```vue
<template>
  <img
    src="https://nuxt.studio/landing.png"
    @load="onLoad"
    @error="onLoad"
    style="width: 50%"
  />
</template>

<script>
export default {
  setup() {
    const onLoad = (e) => {
      console.log("looooaded", e);
    };
    return {
      onLoad,
    };
  },
};
</script>
```

**Reply by gregorvoinov** [reply]
hmm have you tested this locally? because it doesn't work there either....

updated the sandbox but @error also doesn't fire an event.

**Reply by TheAlexLichter** [reply]
I tested it via [my StackBlitz](https://stackblitz.com/edit/nuxtframeworkdiscussions10131?file=app.vue).

<img width="1782" alt="image" src="https://user-images.githubusercontent.com/640208/212853579-69dd4794-a8b7-40d2-9713-c5823da536a0.png">

---

Though, after testing a bit more, it seems it only is triggered after HMR, so only "on client-side" 🤔

**Reply by TheAlexLichter** [reply]
This might be a vue3 bug, see https://stackblitz.com/edit/vuessr-nuxtframeworkdiscussions10131?file=src%2Fpages%2FHome.vue

**Reply by gregorvoinov** [reply]
hmm but there the event gets fired... or do you mean something else?

**Reply by TheAlexLichter** [reply]
The console stays empty for me there after page reload 	🤔

<img width="1774" alt="image" src="https://user-images.githubusercontent.com/640208/212858415-e92a2539-594c-4224-916c-b7f6d571041d.png">

**Reply by gregorvoinov** [reply]
I get the log in the console

![image](https://user-images.githubusercontent.com/9499590/212860662-d4ca6c38-028e-4d8b-8d45-549dc38db128.png)

**Reply by TheAlexLichter** [reply]
Fun 😅
Guess that's Chrome/FF difference here.

**Reply by gregorvoinov** [reply]
ok ff blocks loading the image because of cross origin. Chrome doesn't, but that means in vue3 it works and the problem is with nuxt3?

**Reply by TheAlexLichter** [reply]
Testing in chrome, the event is triggered too via my Nuxt stackblitz 👀

<img width="1784" alt="image" src="https://user-images.githubusercontent.com/640208/212878390-292852b0-64e6-4776-8d82-0b1fe8ce54d8.png">

**Reply by gregorvoinov** [reply]
ok created a new project and there the load event works...  not sure what's running wrong in my current project, but the error must be on my side... sry for wasting your time.

**Reply by TheAlexLichter** [reply]
No worries at all 😋
